apps/builder/app/builder/shared/code-editor.tsx

```diff
@@ -1,386 +1,122 @@
+import { forwardRef, useMemo, type ComponentProps, useEffect } from "react";
```

Comment:
If put base code editor here the diff would not blow up

Reply:
I have no idea what you mean

Reply:
splitting code editor stuff into code-editor-base made diff unreadable

Reply:
I would just keep both components in single module

Reply:
I didn't split it now, it was before

Reply:
I think git didn't recognize renaming because it also had changes

Reply:
because after remaining you created a module with the same name